src/components/materialization/create/CollectionConfig.tsx

```diff
@@ -15,6 +19,22 @@ function CollectionConfig() {
         createStoreSelectors.details.connectorTag
     );
     const collections = useCreationStore(creationSelectors.collections);
+    const prefillCollections = useCreationStore(
+        creationSelectors.prefillCollections
+    );
+
+    const [searchParams] = useSearchParams();
+    const specID = searchParams.get(
+        routeDetails.materializations.create.params.specID
+    );
+
+    const { liveSpecs } = useLiveSpecsExt(specID);
+
+    useEffect(() => {
+        if (liveSpecs.writes_to.length > 0) {
+            prefillCollections(liveSpecs.writes_to);
+        }
+    }, [liveSpecs, prefillCollections]);
```

Comment:
Instead of passing collections into materializations it is easier to just pass the spec id that can then pull the list of collections.